### ISSUE TITLE
make HTC Julia jobs guide

### ISSUE THREAD

**Issue by jmvera255** [CHTC/chtc-website-source#127]
I think we still have a bit to learn about recommendations for using Julia but getting a guide started will be a good starting point

**Comment by ChristinaLK** [comment]
Sample job script: 
```
#!/bin/bash

tar xzf julia-1.4.0-linux-x86_64.tar.gz
export PATH=$PWD/julia-1.4.0/bin:$PATH

## for packages
# tar -xzf pkgdir.tar.gz
# export JULIA_DEPOT_PATH=$PWD/pkgdir

julia vol.jl
```

**Comment by ChristinaLK** [comment]
Closed by #192
